standard/standard_2.php

```diff
@@ -993,7 +993,7 @@ function srand ($seed = null, $mode = MT_RAND_MT19937) {}
 function getrandmax () {}
 
 /**
- * Generate a better random value
+ * Generate a random value via the Mersenne Twister Random Number Generator
```

Comment:
Please, add description for parameters

Reply:
In this PR we are going to add default values, but according to description `max` default value is  `mt_getrandmax()` so it'll be better to mention it in docs

Reply:
Done